zebra/zebra_nb.c

```diff
@@ -34,15 +34,17 @@ const struct frr_yang_module_info frr_zebra_info = {
 		{
 			.xpath = "/frr-zebra:zebra/ip-forwarding",
 			.cbs = {
-				.modify = zebra_ip_forwarding_modify,
-				.destroy = zebra_ip_forwarding_destroy,
+				//.modify = zebra_ip_forwarding_modify,
+				//.destroy = zebra_ip_forwarding_destroy,
+				.get_elem = zebra_ip_forwarding_get_elem,
 			}
 		},
 		{
 			.xpath = "/frr-zebra:zebra/ipv6-forwarding",
 			.cbs = {
-				.modify = zebra_ipv6_forwarding_modify,
-				.destroy = zebra_ipv6_forwarding_destroy,
+				//.modify = zebra_ipv6_forwarding_modify,
```

Comment:
Can we drop these lines at all then?

Reply:
I'm planning on a followup commit to address this exact problem.

Reply:
I had Donna move the get function / yang to the state container